src/test/java/net/oauth/jsontoken/JsonTokenParserTest.java

```diff
@@ -435,21 +435,20 @@ private JsonToken getJsonTokenWithTimeRange(Instant issuedAt, Instant expiration
     return checkToken;
   }
 
+  // deserializes the token string such that tokenStrings without signatures are allowed
```

Comment:
nit: Use Javadoc format for describing the overall purpose or behavior of a method. The regular "//" comment format can be used for implementation comments on specific lines (https://google.github.io/styleguide/javaguide.html#s7.3.4-javadoc-non-required)

Reply:
Changed.